android/src/main/java/io/rownd/android/views/RowndWebView.kt

```diff
@@ -265,6 +294,12 @@ class RowndJavascriptInterface(private val parentWebView: RowndWebView) {
             MessageType.CloseHubView -> {
                 parentWebView.dismiss?.invoke()
             }
+
+            MessageType.tryAgain -> {
+                parentWebView.dismiss?.invoke()
```

Comment:
It would be good if we could just reload the original URL in the existing webview instead of closing and re-opening.

Reply:
Updated `webView.loadUrl(Rownd.config.hubLoaderUrl())`